gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcTransportChannel.java

```diff
@@ -100,17 +106,23 @@ public void close() {
   }
 
   public static Builder newBuilder() {
-    return new AutoValue_GrpcTransportChannel.Builder();
+    return new AutoValue_GrpcTransportChannel.Builder().setUseDirectPath(false);
   }
 
   public static GrpcTransportChannel create(ManagedChannel channel) {
     return newBuilder().setManagedChannel(channel).build();
   }
 
+  public static GrpcTransportChannel create(ManagedChannel channel, boolean canUseDirectPath) {
```

Comment:
Since we already have a buider, I don't think we need to create another helper method.

Reply:
good point! let me switch to builder.